### DISCUSSION TITLE
Will there be a new version of Hover Zoom+ for Google Chrome?

### DISCUSSION THREAD

**Discussion by SwiftyMagee** [extesy/hoverzoom#1511] [General]
Google just recently updated their Google Chrome browser, disabling many of the older extensions. I'm not well-versed with the issue but I think it has to do with Google requiring a new version of extension manifests. All I'm really sure of is that when I opened Chrome about 10 minutes ago, a few of my extensions - including this one - have been disabled.

I love Hover Zoom+ and have been using it for I don't know how long. My question is, will this extension eventually be updated for Google Chrome's new requirements for extensions, or will there be a newer version of the app released ("Hover Zoom++", maybe?). If not, does anyone know of a good alternative to Hover Zoom+?

**Comment by SarreqTeryx** [comment]
it would be nice if this were Manifest v3 compatible, since Google's being such a pain in the butt with it.  I was using Imagus, and it doesn't look like that's being developed anymore¸as far as I can find.
